packages/node_modules/@webex/react-redux-spark/src/actions.test.js

```diff
@@ -37,6 +37,15 @@ describe('sdk actions', () => {
       });
   });
 
+  it('should unregister this device with spark', () => {
```

Comment:
Glad you were able to add a test in a clean manner. Good job!

Reply:
Thanks!